janus_core/cli/phonons.py

```diff
@@ -206,6 +225,24 @@ def phonons(
         raise ValueError("'fmax' must be passed through the --fmax option")
     minimize_kwargs["fmax"] = fmax
 
+    try:
+        supercell = [int(x) for x in supercell.split()]
+    except ValueError as exc:
+        raise ValueError(
+            "Please pass lattice vectors as integers in the form '1 2 3'"
```

Comment:
This doesn't necessarily need the full set of options again, but could maybe be a little more descriptive. Perhaps

```suggestion
            "Please pass lattice vectors as space-separated integers in quotes. "
            "For example, '1 2 3'"
```

Reply:
Done!